ui/app/helpers/transactions.util.js

```diff
@@ -80,6 +80,8 @@ export async function getTransactionActionKey (transaction, methodData) {
         return APPROVE_ACTION_KEY
       case TOKEN_METHOD_TRANSFER_FROM:
         return TRANSFER_FROM_ACTION_KEY
+      default:
+        return
```

Comment:
Can we make `undefined` explicit here? (i.e. `return undefined`)

Reply:
Also now might be a good time to add a doc comment to `getTransactionActionKey` detailing its args and return value